app/lobby/lobby.js

```diff
@@ -0,0 +1,67 @@
+const { randomInt } = require("../utils.js");
+const { PLAYERS_PER_GAME, ROLES } = require("../constants.js");
+
+const LOBBIES = {};
```

Comment:
`lobby.js` has a collection of Lobbies. That's pretty confusing.

How are we defining a "lobby"? We have "The Lobby", which is the homepage of our site where users can join other users in individual "lobbies". I propose we rename one of these concepts. What if we call a pairing of two players a "match"?
A Match object could have not only a list of its current clients, but also game state and game logic. These matches would be in some dictionary object, where the keys are the corresponding unique matchIDs. When a request is received from any user, the matchID is looked up, and the request data is forwarded into the Match object to handle it.
The Match might have functions like `isClientInMatch?(clientToken)` and `getClientRole(clientToken)` or whatever else is needed for an instance of the game.